application/src/main/java/org/togetherjava/tjbot/commands/github/GitHubReference.java

```diff
@@ -135,7 +136,7 @@ protected MessageEmbed generateReply(GHIssue issue) throws UncheckedIOException
 
             // Format: 9 Oct, 2023
             String dateOfCreation = "%d %s, %d".formatted(createdAt.get(Calendar.DATE),
-                    months.get(createdAt.get(Calendar.MONTH)), createdAt.get(Calendar.YEAR));
+                    MONTHS.get(createdAt.get(Calendar.MONTH)), createdAt.get(Calendar.YEAR));
```

Comment:
`Calender` is from the old legacy date api. use `java.time` instead. u can also delete that `MONTHS` list entirely.

```java
DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd MMM, yyyy");
...
String text = ZonedDateTime.now(ZoneOffset.UTC).format(FORMATTER);
```

Reply:
done